steps/saucelabs-espresso-app-automate/0.1.5/step.yml

```diff
@@ -0,0 +1,73 @@
+title: SauceLabs App Automate
+summary: |
+  Upload an app apk and an Espresso test apk to SauceLabs and run tests with App Automate.
+description: |
+  This step uploads a provided app apk and  test apk to SauceLabs.  It then will execute the Espresso Tests for the provided apks in SauceLabs app automate.
+website: https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate
+source_code_url: https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate
+support_url: https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate/issues
+published_at: 2019-12-03T18:14:38.369017+01:00
+source:
+  git: https://github.com/moatazeldebsy/bitrise-step-saucelabs-espresso-app-automate.git
+  commit: dae3aa896006e1b2c9a0ea358be8df371413c91b
+host_os_tags:
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- app_apk_path: $BITRISE_APK_PATH
+  opts:
+    description: |
+      The .apk file for your app
+    is_expand: true
+    is_required: true
+    summary: Path to the app .apk file
+    title: App apk path
+- opts:
+    description: |
+      The test .apk file for your app, likely generated by the Android Build for UI Testing step
+    is_expand: true
+    is_required: true
+    summary: Path to the espresso test .apk file
+    title: Test suite apk path
+  test_apk_path: $BITRISE_TEST_APK_PATH
+- opts:
+    description: ""
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Your username for saucelabs
+    title: SauceLabs username
+  saucelabs_username: null
+- opts:
+    description: ""
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Your access key for saucelabs
+    title: SauceLabs access key
+  saucelabs_access_key: null
+- opts:
+    description: |
+      The Device and OS version you want to run your tests on, with quotes. Acceptable format example: Google Pixel GoogleAPI Emulator,platformVersion=7.0
+      A list of available device and OS combinations can be found [here](https://saucelabs.com/platform/supported-browsers-devices)
+    is_expand: true
+    is_required: true
+    summary: The desired test device
+    title: Device Name
+  saucelabs_device_name: ""
+- opts:
+    description: |
+      Set this parameter if you want to specify a Sauce Labs datacenter Options: "us-west-1" , "eu-central-1"
+    is_expand: false
```

Comment:
Why this parameter is not expandable?

Reply:
@koral-- because it will include only one option `us-west-1` or  `eu-central-1`

Reply:
OK, so it should contain `value_options`.
https://devcenter.bitrise.io/bitrise-cli/step-inputs/

Reply:
Shall I close this PR a create a new one with new tag or the same

Reply:
@moatazeldebsy yes please, it would end up in a more easy way to configure for the users! 🙂

Reply:
sure will do